### DISCUSSION TITLE
Welcome to System Log Parser Discussions!

### DISCUSSION THREAD

**Discussion by AaronPLopez** [Esri/SystemLogParser#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by azinsharaf** [comment]
My name is Azin. A GIS Analyst in Wood Rodgers, who is using system log parser as a way to monitor the performance of our servers.
